File: backend/src/BiteRight.Application/Queries/Units/Search/SearchResponse.cs
// # ==============================================================================
// # Solution: BiteRight
// # File: SearchResponse.cs
// # Author: Łukasz Sobczak
// # Created: 12-02-2024
// # ==============================================================================

#region

using BiteRight.Application.Dtos.Common;
using BiteRight.Application.Dtos.Units;

#endregion

namespace BiteRight.Application.Queries.Units.Search;

// ReSharper disable once NotAccessedPositionalProperty.Global
public record SearchResponse(PaginatedList<UnitDto> Units);

File: mobile/app/src/main/java/com/sobczal2/biteright/data/api/responses/units/SearchResponse.kt
package com.sobczal2.biteright.data.api.responses.units

import com.google.gson.annotations.SerializedName
import com.sobczal2.biteright.dto.common.PaginatedList
import com.sobczal2.biteright.dto.units.UnitDto

data class SearchResponse(
    @SerializedName("units") val units: PaginatedList<UnitDto>
)